add-ons/ansible-service-broker/README.md

```diff
@@ -0,0 +1,70 @@
+Ansible Service Broker Addon
+======================
+
+This addon installs the [Ansible Service Broker](https://github.com/openshift/ansible-service-broker)
```

Comment:
We refer to add-ons with hyphenation within the documentation -- please do so throughout this document as well. :)